Block/Adminhtml/System/Config/Form/Field/Version.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rvvup\Payments\Block\Adminhtml\System\Config\Form\Field;
+
+use Magento\Backend\Block\Template\Context;
+use Magento\Config\Block\System\Config\Form\Field;
+use Magento\Framework\Component\ComponentRegistrarInterface;
+use Magento\Framework\Data\Form\Element\AbstractElement;
+use Magento\Framework\Component\ComponentRegistrar;
+use Magento\Framework\Exception\FileSystemException;
+use Magento\Framework\Exception\ValidatorException;
+use Magento\Framework\Filesystem\Directory\ReadFactory;
+
+class Version extends Field
+{
+    /**
+     * @var ComponentRegistrarInterface
+     */
+    private ComponentRegistrarInterface $componentRegistrar;
+
+    /**
+     * @var ReadFactory
+     */
+    private ReadFactory $readFactory;
+
+    /**
+     * @param Context $context
+     * @param ComponentRegistrarInterface $componentRegistrar
+     * @param ReadFactory $readFactory
+     * @param array $data
+     */
+    public function __construct(
+        Context $context,
+        ComponentRegistrarInterface $componentRegistrar,
+        ReadFactory $readFactory,
+        array $data = []
+    ) {
+        $this->componentRegistrar = $componentRegistrar;
+        $this->readFactory = $readFactory;
+        parent::__construct(
+            $context,
+            $data
+        );
+    }
+
+    protected function _getElementHtml(AbstractElement $element)
+    {
+        $element->setDisabled('disabled');
+        $element->setValue($this->getRvvupPluginVersion());
+
+        return $element->getElementHtml();
+    }
+
+    /**
+     * @return string
+     * @throws FileSystemException
+     * @throws ValidatorException
+     */
+    private function getRvvupPluginVersion(): string
```

Comment:
Is it possible to write some tests for this? 

I'm also wondering if its every possible that there is no composer.json? In that case would an exception be thrown as we try to read the file or will it correctly return an empty string?

Reply:
In case it won't exist in compose (like in our local environment) it will return an empty string. 
I will think about adding tests, thank you